Assets/Scripts/ProceduralGeneration.cs

```diff
@@ -111,6 +111,8 @@ private void Start()
     {
         ui = FindObjectOfType<UIManager>();
         ui.SetLevelStatus(UIManager.LevelStatus.Loading);
+        transform.localScale = levelDimensions;
```

Comment:
What is this for?

Reply:
This is needed so that the trigger collider we use to sic the aliens on you represents the boundaries of the level